monai/transforms/lazy/functional.py

```diff
@@ -23,10 +23,10 @@
     resample,
 )
 
-__all__ = ["apply"]
+__all__ = ["apply_transforms"]
 
 
-def apply(data: Union[torch.Tensor, MetaTensor], pending: Optional[list] = None):
```

Comment:
for some reason this is conflicting with an existing `apply` function, so I'm renaming it to `apply_transforms`.

otherwise the autofix is making this change:
```diff
-        result = apply(tensor, pending_transforms)
+        result = tensor(*pending_transforms)
```

cc @atbenmurray